.storybook/pages/AlongDemo/AlongDemo.stories.ts

```diff
@@ -1,5 +1,5 @@
 import type { StoryObj, Meta } from '@storybook/react';
-import { within } from '@storybook/testing-library';
+import { userEvent, within } from '@storybook/testing-library';
```

Comment:
https://github.com/storybookjs/testing-library/issues/26 is tracking supporting user-event 14 in @storybook/testing-library.